warehouse/schema/schema_test.go

```diff
@@ -111,27 +114,46 @@ func TestSchema(t *testing.T) {
 	})
 	ctx := context.Background()
 
-	t.Run("SyncRemoteSchema", func(t *testing.T) {
-		schema, err := sch.getSchema(ctx)
-		require.NoError(t, err)
-		require.Equal(t, model.Schema{}, schema)
-		require.True(t, sch.IsWarehouseSchemaEmpty(ctx))
-
-		_, err = sch.SyncRemoteSchema(ctx, nil, 0)
-		require.NoError(t, err)
-		schema, err = sch.getSchema(ctx)
-		require.NoError(t, err)
-		require.Equal(t, model.Schema{
+	t.Run("Test IsSchemaEmpty", func(t *testing.T) {
```

Comment:
We don't need `Test` prefix for the test name?
```suggestion
	t.Run("IsSchemaEmpty", func(t *testing.T) {
```